packages/hdwallet-native/src/ethereum.ts

```diff
@@ -53,32 +54,35 @@ export function MixinNativeETHWallet<TBase extends core.Constructor<NativeHDWall
   return class MixinNativeETHWallet extends Base {
     readonly _supportsETH = true;
 
-    #ethSigner: ethers.Signer | undefined;
+    #ethSigner: SignerAdapter | undefined;
 
     async ethInitializeWallet(masterKey: Isolation.Core.BIP32.Node): Promise<void> {
       const rootNode = await Isolation.Adapters.BIP32.create(masterKey);
-      const isolatedSigner = await rootNode.derivePath(ethers.utils.defaultPath);
-      this.#ethSigner = await Isolation.Adapters.Ethereum.create(isolatedSigner.node);
+
+      this.#ethSigner = await Isolation.Adapters.Ethereum.create(rootNode);
     }
 
     ethWipe() {
       this.#ethSigner = undefined;
     }
 
     async ethGetAddress(msg: core.ETHGetAddress): Promise<string | null> {
-      if (!_.isEqual(msg.addressNList, core.bip32ToAddressNList("m/44'/60'/0'/0/0"))) {
-        throw new Error("path not supported");
+      if (msg.addressNList.length < 5) {
```

Comment:
wondering if we should just omit this check and allow addresses to be derived in non standard ways if desired. we don't really enforce this anywhere else either

Reply:
yep good point